src/components/smart_objects/include/smart_objects/enum_schema_item.h

```diff
@@ -128,15 +128,17 @@ class TEnumSchemaItem : public CDefaultSchemaItem<EnumType> {
       const std::vector<ElementSignature>& signatures,
       const utils::SemanticVersion& MessageVersion);
 
+  bool filterInvalidEnums(SmartObject& Object,
```

Comment:
Please add method description of params and return value

Reply:
It's in the `schema_item.h` header, we should only have it in one place